.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
```

Comment:
So this would be head of master wouldnt it if we are triggering on release?

Reply:
When we create a release,it tags the commit with the release name.So we can choose it to be created for a commit in any branch

Reply:
Changed the workflow to manual trigger